test/extended/images/helper.go

```diff
@@ -24,8 +24,8 @@ func GetImageLabels(c client.ImageStreamImageInterface, imageRepoName, imageRef
 }
 
 // ModifySourceCode will modify source code in the pod of the application
-// according to the sed script.
-func ModifySourceCode(oc *exutil.CLI, selector labels.Selector, sedScript, file string) error {
+// according to the provided command.
+func ModifySourceCode(oc *exutil.CLI, selector labels.Selector, cmd []string) error {
```

Comment:
I think array is more consistent with rest of our code (and code everywhere else in the universe), plus it frees you from too much quoting when everything is in a string.